src/routes/+layout.server.ts

```diff
@@ -161,6 +163,7 @@ export const load: LayoutServerLoad = async ({ locals, depends }) => {
 			email: locals.user.email,
 		},
 		assistant,
+		avatarGeneration: ASSISTANTS_GENERATE_AVATAR === "true" && TEXT_TO_IMAGE_MODEL !== "",
```

Comment:
```suggestion
		avatarGeneration: ASSISTANTS_GENERATE_AVATAR === "true" && !TEXT_TO_IMAGE_MODEL,
```

I think ts string truthiness would work fine here

Reply:
I think it should have been `!!`  but it works :grin: